tests/e2e/main.go

```diff
@@ -298,6 +325,42 @@ var rootCmd = &cobra.Command{
 			return err
 		}
 
+		// create aws creds secret
+		secretData, err = ioutil.ReadFile(filepath.Join(assetsPath, "manifests/aws-credentials.yaml"))
+		if err != nil {
+			glog.Fatalf("Error reading %#v", err)
+		}
+
+		secretObj, _, err = decode([]byte(secretData), nil, nil)
+		if err != nil {
+			glog.Fatalf("Error decoding %#v", err)
+		}
+		secret = secretObj.(*apiv1.Secret)
+		if err := createSecret(testConfig, secret); err != nil {
+			return err
+		}
+
+		// create ign config secret
+		secretData, err = ioutil.ReadFile(filepath.Join(assetsPath, "manifests/ign-config.yaml"))
+		if err != nil {
+			glog.Fatalf("Error reading %#v", err)
+		}
+		secretObj, _, err = decode([]byte(secretData), nil, nil)
+		if err != nil {
+			glog.Fatalf("Error decoding %#v", err)
+		}
+		secret = secretObj.(*apiv1.Secret)
+		if err := createSecret(testConfig, secret); err != nil {
+			return err
+		}
+
+		awsClient, err := NewClient(testConfig.KubeClient, awsCredentialsSecretName, targetNamespace, region)
```

Comment:
What about the import the `NewClient` from `https://github.com/openshift/cluster-api-provider-aws/blob/master/cloud/aws/client/client.go` instead?

Reply:
This might not be trivial as aws actuator is currently forked but keeps original path references.
Is there a way to choose a directory with dep? should we consider update path references to openshift

Reply:
Let's import the openshift fork of the repo everytime we import the aws actuator code. So there is no confusion with the upstream repo.